types/gloas/execution_payload_envelope.yaml

```
@@ -0,0 +1,45 @@
+Gloas:
+  ExecutionPayloadEnvelope:
+    type: object
+    description: "The [`ExecutionPayloadEnvelope`](https://github.com/ethereum/consensus-specs/blob/00d531949b1f30516979b60ddd2a411e7f388299/specs/gloas/beacon-chain.md#executionpayloadenvelope) object from the CL gloas spec."
+    required: [payload, execution_requests, builder_index, beacon_block_root, slot, blob_kzg_commitments, state_root]
+    properties:
+      payload:
+        $ref: "../deneb/execution_payload.yaml#/Deneb/ExecutionPayload"
+      execution_requests:
+        $ref: "../electra/execution_requests.yaml#/Electra/ExecutionRequests"
+      builder_index:
+        allOf:
+          - $ref: "../primitive.yaml#/Uint64"
+          - description: "Index of the builder that created this execution payload"
```

the usage of `allOf` is pretty unnecessary

this can simplify be defined like this
```suggestion
      builder_index:
        $ref: "../primitive.yaml#/Uint64"
        description: "Index of the builder that created this execution payload"
```

---

ah, yep, much simpler. updated here and all other instances I found: https://github.com/ethereum/beacon-APIs/pull/552/commits/900273e731f7245395e0c657710b8f1396f954f9